pytravharv/store.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from datetime import datetime
+from typing import List
+
+from pyrdfj2 import J2RDFSyntaxBuilder
+from pyrdfstore import RDFStore
+from rdflib import Graph
+from rdflib.plugins.sparql.processor import Result
+
+log = logging.getLogger(__name__)
+
+
+class TargetStoreAccess:
+
+    def __init__(self, target: RDFStore, qryBuilder: J2RDFSyntaxBuilder):
+        self._target = target
+        self._qryBuilder = qryBuilder
+
+    def select_subjects(self, sparql) -> List[str]:
+        result: Result = self._target.select(sparql)
+        # if result is (400, 'HTTP Error 400: ', None) then return empty list
+        if result == (400, "HTTP Error 400: ", None):
```

Comment:
currently the rdfstore implementation has an assertion on board to guarantee the actual response is not an error

you can go for try-except those, but I would advise against silencing the issue like happens here, also do not `print()` but `log`